src/d3d9/d3d9_swapchain.cpp

```diff
@@ -748,6 +748,39 @@ namespace dxvk {
   }
 
 
+  void D3D9SwapChainEx::SetCursorTexture(UINT Width, UINT Height, uint8_t* pCursorBitmap) {
+      VkExtent2D cursorSize = { uint32_t(Width), uint32_t(Height) };
+
+      m_parent->EmitCs([
```

Comment:
You don't need to use CS for this, as long as we know the blitter exists it's safe to call directly.

In fact, using CS might be a problem because I don't see anything keeping the source pointer alive.

Reply:
Alright, called the bitter directly here. It was kept alive by an in-struct vector bitmap buffer, however since that's not really necessary I've dropped that part as well.